lib/logger/lib/logger/cli_formatter.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Logger.CLIFormatter do
```

Comment:
Don't need, same as with `group_leader` in `ExUnit.Test` struct, I've left them to clean up later.